File: golang/sample_test.go
package tdd

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func TestSay(t *testing.T) {
	// when
	actual := Say("Hello!")

	// then
	assert.Equal(t, "Hello! TDD BootCamp!!", actual, "they should be equal")
}

File: ruby/test/sample_test.rb
require 'test/unit'
require 'sample'

class TC_Sample < Test::Unit::TestCase
  def test_say
    # given
    obj = Sample.new

    # when
    actual = obj.say

    # then
    assert_equal('Hello TDD BootCamp!', actual)
  end
end
